src/components/GordonDialogBox/GordonDialogBox.module.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../vars';
 
-.gc360_gordondialogbox {
+.gc360_gordondialogbox .gc360_gordondialogbox {
```

Comment:
Rather than bumping the specificity of the entire dialog (which would make it harder for anything styling the dialog to override it), wouldn't it be better to override the underlying MUI style?

If you change `&_content {` on line 12 to `&_content :global(MuiDialogTitle-root)`, it should override that specific style.

Also, I believe that `.gc360_gordondialogbox .gc360_gordondialogbox` would not work because it selects an element with the class `.gc360_gordondialogbox` **inside** an element with the class `.gc360_gordondialogbox`, right? Shouldn't it be `.gc360_gordondialogbox.gc360_gordondialogbox` instead?